### PR TITLE
Additional content updates

### PR DESCRIPTION
**Pull request by stiegerb**
Opening this for additional content updates to be discussed. 

- Added CMSSW version fields for CRAB3 jobs (`CMSSWVersion` (e.g. 8_1_3_patch1), `CMSSWSubVersion` (e.g. 8_1_X), `CMSSWSubSubVersion` (e.g. 8_X_X)
- Made sure `CMS_JobType` is always filled meaningfully
- Made sure `CRAB_AsyncDest` is always filled meaningfully
- Made sure `CMSPrimaryDataTier`, `CMSPrimaryPrimaryDataset`, `CMSPrimaryProcessedDataset` are always filled meaningfully
- Made sure `MachineAttrMJF_JOB_HS06_JOB0` is not filled if it can't be calculated
- Made sure `RequestCpus` is always filled

### CODE REVIEW

**Comment by stiegerb** [comment]
Note that we thought we fixed `MachineAttrMJF_JOB_HS06_JOB0` in #12, but it remains filled as 'Unknown' there if it was already unknown before. This will now remove it in any case where it can't be filled with a meaningful float.

**Comment by stiegerb** [comment]
@bbockelm If you're ok with these, you could also merge them before #18. I've tested them already for MONIT, so should be ok unless you disagree with the logic somewhere.

File: src/htcondor_es/convert_to_json.py [hunk 1]
```diff
@@ -798,26 +819,35 @@ def convert_to_json(ad, cms=True, return_dict=False, reduce_data=False):
     if ('Chirp_WMCore_cmsRun_ExitCode' in result) and (result.get('ExitCode', 0) == 0):
         result['ExitCode'] = result['Chirp_WMCore_cmsRun_ExitCode']
 
+    # Parse CRAB3 information on CMSSW version
+    result['CMSSWVersion'] = 'Unknown'
+    result['CMSSWSubVersion'] = 'Unknown'
```
Comment (bbockelm) [inline src/htcondor_es/convert_to_json.py]: I don't understand the motivation behind the Sub and SubSub version?

Comment (stiegerb) [reply src/htcondor_es/convert_to_json.py]: I'll rename them to `CMSSWMajorVersion`, and `CMSSWReleaseSeries`

**Comment by bbockelm** [comment]
LGTM.  Merging.

**Pull request merged**